app/db/models/app/users.py

```diff
@@ -26,13 +26,15 @@ class Organisation(Base):
     organisation_type = sa.Column(sa.String)
 
 
-class AdminOrganisation(Base):
+class OrganisationUser(Base):
```

Comment:
Arguably could be called OrganisationAdmin?

Reply:
Honestly, this was just written with the expectation that we may want different roles in the future.